extension/ezjscore/classes/ezjscpacker.php

```diff
@@ -461,6 +462,7 @@ static function packFiles( $fileArray, $subPath = '', $fileExtension = '.js', $p
 
             $content .= "/* start: $file */\r\n";
             $content .= $fileContent;
+            if ($isJS) {$content .= ';';}
```

Comment:
Hello @sterrien 

I think we are actually looking for the following (instead), in terms of cs:

    if ( $isJS )
    {
        $content .= ';';
    }

This spacing and bracket placement cs also applies to the rest of your additions. I would suggest refactoring all your additions to more closely match these conventions as it increases readability and more closely matches the existing code.

Cheers,
Brookins Consulting